kalite/main/api_resources.py

```diff
@@ -269,6 +270,7 @@ def __init__(self, lang_code="en", **kwargs):
         self.selected_language = lang_code
         if self.description == "None":
             self.description = ""
+        self.source = settings.CHANNEL
```

Comment:
Can you add a TODO (BLOCKER) to change this for 0.14? We will need to make this happen dynamically when we have multiple channels active at once.